src/components/forms/AddServiceForm.js

```diff
@@ -79,8 +79,11 @@ function AddServiceForm({
           >
             <Select size="large" placeholder="Select Service Type">
               {serviceTypes.map(item => (
-                <Select.Option key={item.id} value={item.id}>
-                  {item.name}
+                <Select.Option
+                  key={item.service_type_id}
+                  value={item.service_type_id}
+                >
+                  {item.service_type_name}
```

Comment:
This seems like this was a needed change for the frontend and was discussed and collaborated on with the backend team. Anytime we change field names, let's make sure we test as much as we can, the entire application that interfaces with the resource at hand (in this case the 'services resource').